firstvoices/backend/tests/test_tasks/test_media_tasks.py

```diff
@@ -11,10 +18,34 @@ class TestThumbnailGeneration:
     )
     def test_thumbnail_generation_started(self, model_factory, model, caplog):
         site = SiteFactory()
-        media_item = model_factory.create(site=site, title="Test media item")
+        media_item = model_factory.create(site=site)
 
         assert (
             f"Task started. Additional info: "
             f"model_name: {model}, instance_id: {media_item.id}." in caplog.text
         )
         assert "Task ended." in caplog.text
+
+    @pytest.mark.django_db
+    @pytest.mark.disable_thumbnail_mocks
+    @pytest.mark.parametrize(
+        "model, model_factory, model_file_factory",
+        [
+            (Image, ImageFactory, ImageFileFactory),
+            (Video, VideoFactory, VideoFileFactory),
+        ],
+    )
+    def test_thumbnail_generation_does_not_update_last_modified(
+        self, model, model_factory, model_file_factory
+    ):
+        site = SiteFactory()
+
+        media_file = model_file_factory.create()
+        media_item = model_factory.create(site=site, original=media_file)
+        original_last_modified = media_item.last_modified
+
+        media_item = model.objects.get(id=media_item.id)
```

Comment:
I think we have all the celery tasks set to run synchronously, so the thumbnail generation would finish running before you set the original_last_modified date. Is that true? If so, a good test would be to add a line in the test calling generate_media_thumbnails for the instance, and then assert that it didn't change the date